### PR TITLE
new README.md proposal

### PR DESCRIPTION
**Pull request by dlaboss**
Clearly direct readers to the main IO page where they can easily find
more information on all aspects of Quarks.  Right now its more work than
it should be for someone to be able to locate the releases/downloads,
faq, ....  The link to IO at the top of the page easily goes unnoticed -
I know! :-)

Remove all of the overview info as it's already covered in the main IO
page and its links.  Avoid maintaining multiple versions of it.

### CODE REVIEW

**Comment by ddebrunner** [comment]
I intentionally put the information there rather than just a link.

Putting a link makes it less likely that people will follow it to find out what the project is about.

I slo think the README should standalone, rather than relying on an different site.

I'm -1 on this change.

**Comment by dlaboss** [comment]
Got it.  I still, based on my experience, believe there's a need/benefit for to better direct users to info related to releases/faq/... from README.  OK with that?

**Comment by dlaboss** [comment]
pushed a commit reverting to the original plus additional refs at the end for review.

File: README.md [hunk 1]
```diff
@@ -62,3 +62,8 @@ Quarks communicates with your back-end systems through the following message hub
 Your back-end systems can also use analytics to interact with and control edge devices. For example:
 * A traffic alert system can send an alert to vehicles that are heading towards an area where an accident occurred
 * A vehicle monitoring system can reduce the maximum engine revs to reduce the chance of failure before the next scheduled service if it detects patterns that indicate a potential problem
+
+See http://quarks-edge.github.io/ for more information on all aspects of Quarks!
+
+Additional information for developing the Quarks runtime can also be found [here](DEVELOPING)
```
Comment (ddebrunner) [inline README.md]: Is DEVELOPING meant to be DEVELOPMENT.md ?

Also is it possible to be more encompassing than "Quarks runtime"?

"Additional information for how to contribute to the development of Quarks can also be found here"

??

**Pull request merged**